middleware/near.js

```diff
@@ -44,6 +55,25 @@ async function checkAccountOwnership(ctx, next) {
     return await next();
 }
 
+
+function getAccount(ctx, accountId) {
+    return new nearAPI.Account(ctx.near.connection, accountId);
```

Comment:
generally should be possible to use `await ctx.near.account()` instead of this method.

Reply:
not sure if that applies here because I need to check account for user provided accountId